gpkit/interactive/plot_sweep1d.py

```diff
@@ -0,0 +1,48 @@
+"Implements plot_sweep1d function"
+import matplotlib.pyplot as plt
+import numpy as np
+from ..exceptions import InvalidGPConstraint
+from ..tools import autosweep_1d
+
```

Comment:
I think that should be an option, but that this is still a useful function...

Scripting of the "this variable is a function of the results of these variables" kind can now be done automatically with `evalfn=` Variable metadata.